Formula/vulkan-extensionlayer.rb

```diff
@@ -0,0 +1,66 @@
+class VulkanExtensionlayer < Formula
+  desc "Layer providing Vulkan features when native support is unavailable"
+  homepage "https://github.com/KhronosGroup/Vulkan-ExtensionLayer"
+  url "https://github.com/KhronosGroup/Vulkan-ExtensionLayer/archive/refs/tags/v1.3.246.tar.gz"
+  sha256 "6ce8e35dd56d46f308f4581a4d0cdfd660b9aa733a4271442365fda422d4227b"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-ExtensionLayer.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "googletest" => :build
+  depends_on "python@3.11" => :build
+  depends_on "vulkan-loader" => :test
+  depends_on "vulkan-tools" => :test
+  depends_on "glslang"
+  depends_on "spirv-headers"
+  depends_on "spirv-tools"
+  depends_on "vulkan-headers"
+
+  on_linux do
+    depends_on "libxcb" => :build
+    depends_on "libxrandr" => :build
+    depends_on "mesa" => :build
+    depends_on "pkg-config" => :build
+    depends_on "wayland" => :build
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+           "-DGLSLANG_INSTALL_DIR=#{Formula["glslang"].prefix}",
+           "-DSPIRV_HEADERS_INSTALL_DIR=#{Formula["spirv-headers"].prefix}",
+           "-DSPIRV_TOOLS_INSTALL_DIR=#{Formula["spirv-tools"].prefix}",
+           "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].prefix}",
+           *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DGLSLANG_INSTALL_DIR=#{Formula["glslang"].prefix}",
                    "-DSPIRV_HEADERS_INSTALL_DIR=#{Formula["spirv-headers"].prefix}",
                    "-DSPIRV_TOOLS_INSTALL_DIR=#{Formula["spirv-tools"].prefix}",
                    "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].prefix}",
                    *std_cmake_args
```
Let's align these as arguments of `cmake` and not `system`.